src/main/resources/config/application.yml

```diff
@@ -153,6 +153,7 @@ xm-config:
 # ===================================================================
 
 application:
+    scheduler-enabled: false
```

Comment:
add like in entity 

`        stream:
            kafka:
                binder:
                    health-timeout: 5 # kafkaHealthCheck timeout when scheduler-enabled: true, in seconds`

Reply:
done